chunky/src/java/se/llbit/chunky/block/MinecraftBlockProvider.java

```diff
@@ -1074,6 +1074,51 @@ private static void addBlocks(Texture texture, String... names) {
     addBlock("bamboo_wall_hanging_sign", (name, tag) -> wallHangingSign(tag, "bamboo"));
     addBlock("cherry_hanging_sign", (name, tag) -> hangingSign(tag, "cherry"));
     addBlock("cherry_wall_hanging_sign", (name, tag) -> wallHangingSign(tag, "cherry"));
+
+    //1.21
+    addBlock("tuff_slab", (name, tag) -> slab(tag, Texture.tuff));
+    addBlock("tuff_stairs", (name, tag) -> stairs(tag, Texture.tuff));
+    addBlock("tuff_wall", (name, tag) -> wall(tag, Texture.tuff));
+    addBlock("polished_tuff", Texture.polishedTuff);
+    addBlock("polished_tuff_slab", (name, tag) -> slab(tag, Texture.polishedTuff));
+    addBlock("polished_tuff_stairs", (name, tag) -> stairs(tag, Texture.polishedTuff));
+    addBlock("polished_tuff_wall", (name, tag) -> wall(tag, Texture.polishedTuff));
+    addBlock("tuff_bricks", Texture.tuffBricks);
+    addBlock("tuff_brick_slab", (name, tag) -> slab(tag, Texture.tuffBricks));
+    addBlock("tuff_brick_stairs", (name, tag) -> stairs(tag, Texture.tuffBricks));
+    addBlock("tuff_brick_wall", (name, tag) -> wall(tag, Texture.tuffBricks));
+    addBlock("chiseled_tuff", Texture.chiseledTuff);
+    addBlock("chiseled_tuff_bricks", Texture.chiseledTuffBricks);
+    for(String s : new String[]{"", "waxed_"}) {
```

Comment:
Nice. Maybe we should do this for the other copper blocks too?

Reply:
The other ones are done in a switch statement, so idk how it would work
![image](https://github.com/chunky-dev/chunky/assets/46458276/9316a285-2db7-4912-adbb-0196bf91485d)